src/inject/dynamic-theme/index.ts

```diff
@@ -406,6 +406,8 @@ export function createOrUpdateDynamicTheme(filterConfig: FilterConfig, dynamicTh
         if (isAnotherDarkReaderInstanceActive()) {
             return;
         }
+        document.documentElement.setAttribute('data-darkreader-mode', filter.engine);
```

Comment:
`filter.engine` will return `dynamicTheme`, which is a bit odd, so let's use just `'dynamic'`.